phper/src/modules.rs

```diff
@@ -51,16 +51,16 @@ unsafe extern "C" fn module_startup(_type: c_int, module_number: c_int) -> c_int
             constant.register(module_number);
         }
 
+        for interface_entity in &module.interface_entities {
```

Comment:
Create interfaces before classes, so that the CE is registered before we try to implement (in the case of a module-provided class implementing a module-provided interface).